src/test/java/de/tum/in/www1/artemis/TextExerciseIntegrationTest.java

```diff
@@ -129,6 +152,37 @@ public void createTextExercise() throws Exception {
         assertThat(newTextExercise.getCourseViaExerciseGroupOrCourseMember().getId()).as("exerciseGroupId was set correctly").isEqualTo(course.getId());
```

Comment:
I would add assertions like these in all test cases, just to absolutely make sure the body is e.g. empty when a bad request gets returned